lib/rollbar/request_data_extractor.rb

```diff
@@ -130,6 +135,11 @@ def rollbar_raw_body_params(rack_req)
       rack_req.body.rewind
     end
 
+    def request_has_json_body?(rack_req)
+      rack_req.env['CONTENT_TYPE'] =~ %r{application/json} ||
+      rack_req.env['ACCEPT']       =~ /\bjson\b/
```

Comment:
This regexp may not work for a typical `ACCEPT` header: `Accept: application/vnd.heroku+json; version=3`